tests/core/gas-strategies/test_time_based_gas_price_strategy.py

```diff
@@ -35,12 +34,12 @@ def _get_block_by_something(method, params):
                 {'gasPrice': 5},
                 {'gasPrice': 50},
             ],
-            'miner': '0xA',
+            'miner': '0x' + 'AA' * 20,
```

Comment:
context for miner hash change?

Reply:
This test provides its own middleware, which did not include any normalization, so previously it didn't matter what got passed back. Now that the normalization is unavoidable, '0xA' throws an error when it tries to convert the address to a checksum address.

Reply:
👍 thanks for the explanation!